homework-submission/react-hyf/server/server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const path = require('path');
+const buildFolder = '../client/build';
+const app = express();
+
+app.use(express.static(path.join(__dirname, buildFolder)));
+
+app.get('/', (req, res) => {
+  res.sendFile(express.static(path.join(__dirname, buildFolder, 'index.html')));
```

Comment:
it should be `/*` otherwise only root url works.
if you now go to `localhost:4000/users` it doesn't work, server handles it.